client/signup/config/flows.js

```diff
@@ -109,6 +109,10 @@ function getDestinationFromIntent( dependencies ) {
 		return `/post/${ siteSlug }`;
 	}
 
+	if ( isFSEActive ) {
```

Comment:
I think there's a slight logic hole here where the user can choose the "Write" intent, ask to "skip to my home" and then end up being redirected to the site editor. If the user is in the test cohort then maybe it should change this to a "skip to site editor" button. But in the mean time should it be this:

```suggestion
	if ( isFSEActive && intent !== 'write' ) {
```